universal-login-relayer/lib/integration/ethereum/DeploymentExecutor.ts

```diff
@@ -32,7 +34,7 @@ export class DeploymentExecutor implements IExecutor<Deployment> {
   }
 
   async execute(deployment: Deployment): Promise<providers.TransactionResponse> {
-    return {hash: 'xyz', wait: async () => {}} as any;
+    return this.walletService.deploy(deployment, deployment.deviceInfo);
```

Comment:
This is the actual change that starts executing deployments after taking them from the queue